overlord/snapstate/backend_test.go

```diff
@@ -912,6 +913,22 @@ func (f *fakeSnappyBackend) SetupComponent(compFilePath string, compPi snap.Cont
 	return &backend.InstallRecord{}, nil
 }
 
+func (f *fakeSnappyBackend) SetupKernelModulesComponent(cpi snap.ContainerPlaceInfo, cref naming.ComponentRef, kernelVersion string, meter progress.Meter) (err error) {
+	meter.Notify("setup-kernel-modules-component")
+	f.appendOp(&fakeOp{
+		op: "setup-kernel-modules-component",
```

Comment:
don't we want to capture a bit more about the parameters? in retrospect we should have done so also for SetupComponent and UndoSetupComponent